plenum/test/batching_3pc/catch-up/test_3pc_paused_during_catch_up.py

```diff
@@ -20,3 +22,18 @@ def test_no_ordering_during_syncup(
     # called while syncing
     sendRandomRequests(wallet1, client, tconf.Max3PCBatchSize)
     looper.runFor(5)
+
+
+def test_sdk_no_ordering_during_syncup(tconf, looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_steward):
```

Comment:
Should we use `sdk_wallet_client1` here?

Reply:
make sense